index.bs

```diff
@@ -1345,8 +1349,15 @@ positive integer that controls the maximum number of distinct
 [=attribution rate-limit record/attribution destination=]) that can create
 [=event-level reports=] per [=attribution rate-limit window=].
 
-<dfn>Max attributions per rate-limit window</dfn> is a positive integer that
-controls the maximum number of attributions for a
+<dfn>Max event attributions per rate-limit window</dfn> is a positive integer that
```

Comment:
Can we use a tuple, similar to "max destinations per rate-limit window"?

Reply:
I think it is more readable in current format compared to the tuple impl (similar to max destinations per rate-limit window).
What do you think?

Reply:
Alternatively, maybe we can still use one constant for both? Unless there's a reason to use different limits?

Reply:
If we use separate variables, please update https://github.com/WICG/attribution-reporting-api/blob/main/params/chromium-params.md as well.